### PR TITLE
[PLT-7268] Paper Block

### PR DESCRIPTION
**Pull request by jordanell**
## Description

1. Switch the `<Paper>` component to use default CSS `display` attribute per component.

### CODE REVIEW

File: packages/riipen-ui/src/components/Paper.jsx [hunk 1]
```diff
@@ -35,7 +35,6 @@ const Paper = props => {
       <style jsx>{`
         .root {
           background-color: ${theme.palette.common.white};
-          display: inline-block;
```
Comment (jordanell) [inline packages/riipen-ui/src/components/Paper.jsx:38]: This is the real fix here. Now this component will render with default `display` depending on its Component type.

Comment (daelynj) [reply packages/riipen-ui/src/components/Paper.jsx:38]: Oh yeah that's perfect

**Review by daelynj — APPROVED** [review]
👌

**Review by imogenem — APPROVED** [review]
Makes sense :)

**Pull request merged**